x-pack/plugins/maps/public/components/layer_panel/settings_panel/settings_panel.js

```diff
@@ -28,14 +32,11 @@ export function SettingsPanel(props) {
     props.updateLabel(props.layerId, label);
   };
 
-  const onMinZoomChange = (event) => {
-    const zoom = parseInt(event.target.value, 10);
-    props.updateMinZoom(props.layerId, zoom);
-  };
-
-  const onMaxZoomChange = (event) => {
-    const zoom = parseInt(event.target.value, 10);
-    props.updateMaxZoom(props.layerId, zoom);
+  const onZoomChange = (event) => {
```

Comment:
The callback is not called with an `event`, but with a `value` that is an array of length 2. Maybe rename to `value`?